src/acceptheaderpayee/handler.js

```diff
@@ -0,0 +1,449 @@
+      // received in this handler is an AppError, then send the requester the correct
+      // payload etc. based on the contents of that AppError.
+      // rq.sendError(url, asyncResponses.serverError, rq.defaultHeaders(requesterName, 'participants'), {logger});
+    }
+  })()
+  return h.response().code(202)
+}
+
+exports.postTransfers = async function (req, h) {
+  const histTimerEnd = Metrics.getHistogram(
+    'sim_request',
+    'Histogram for Simulator http operations',
+    ['success', 'fsp', 'operation', 'source', 'destination']
+  ).startTimer()
+
+  Logger.debug(`[cid=${req.payload.transferId}, fsp=${req.headers['fspiop-source']}, source=${req.headers['fspiop-source']}, dest=${req.headers['fspiop-destination']}] ~ Simulator::api::payee::postTransfers - START`)
+
+  const metadata = `${req.method} ${req.path} ${req.payload.transferId}`
+  Logger.info(`IN PAYEEFSP:: received: ${metadata}.`)
+
+  if (!transfersFulfilResponseDisabled) {
+    // Saving Incoming request
+    const incomingRequest = {
+      headers: req.headers,
+      data: req.payload
+    }
+    requests.set(req.payload.transferId, incomingRequest)
+
+    const url = transfersEndpoint + '/transfers/' + req.payload.transferId
+    const fspiopUriHeader = `/transfers/${req.payload.transferId}`
+    try {
+      const transfersResponse = {
+        fulfilment: transfersFulfilment,
+        completedTimestamp: new Date().toISOString(),
+        transferState: 'COMMITTED'
+      }
+      const protectedHeader = {
+        alg: 'RS256',
+        'FSPIOP-Source': `${req.headers['fspiop-destination']}`,
+        'FSPIOP-Destination': `${req.headers['fspiop-source']}`,
+        'FSPIOP-URI': `/transfers/${req.payload.transferId}`,
+        'FSPIOP-HTTP-Method': 'PUT',
+        Date: ''
+      }
+      const fspiopSignature = {
+        signature: signature,
+        protectedHeader: `${base64url.encode(JSON.stringify(protectedHeader))}`
+      }
+      const opts = {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/vnd.interoperability.transfers+json;version=1.0',
+          'FSPIOP-Source': req.headers['fspiop-destination'],
+          'FSPIOP-Destination': req.headers['fspiop-source'],
+          Date: new Date().toUTCString(),
+          'FSPIOP-Signature': JSON.stringify(fspiopSignature),
+          'FSPIOP-HTTP-Method': 'PUT',
+          'FSPIOP-URI': fspiopUriHeader,
+          traceparent: req.headers.traceparent ? req.headers.traceparent : '',
+          tracestate: req.headers.tracestate ? req.headers.tracestate : ''
+        },
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        }),
+        data: JSON.stringify(transfersResponse)
+      }
+
+      Logger.info(`Executing PUT: [${url}], HEADERS: [${JSON.stringify(opts.headers)}], BODY: [${JSON.stringify(transfersResponse)}]`)
+      const res = await request(url, opts)
```

Comment:
I dont believe that this would work. axios requires the url to be sent in the opts. therefore your call should look like 
```
const opts = {
    url: '',
    method: 'PUT',
    headers: [],
    ....
}
const res = await request(opts)
```